server/controllers/authController.js

```diff
@@ -1,13 +1,14 @@
-const loggingInUsers = require('../utilities/loggingInUsers');
+const loggingInUsers = require('../utilities/security/token/loggingInUsers');
+const loginUserAfterSignup = require('../utilities/security/token/loginUserAfterSignup');
 const { promisify } = require('util');
 const jwt = require('jsonwebtoken');
 const User = require('../models/usermodel');
 const bcrypt = require('bcryptjs');
-const sendEmail = require('../utilities/email');
-const generateResetToken = require('../utilities/generateResetToken');
-const encryptResetToken = require('../utilities/encryptResetToken');
-const tokenHasExpired = require('../utilities/tokenHasExpired');
-const sendErrorMessage = require('../utilities/sendErrorMessage');
+const sendEmail = require('../utilities/accessories/email');
```

Comment:
Why did you name this `accessories`?

Reply:
The files were becoming too much in one folder. I needed to organize them into separate folders. Please, can you suggest a better name?

Reply:
I'll change the name.

Reply:
The name has been changed.